registration/api/v2/serializers.py

```diff
@@ -367,6 +367,7 @@ class OpenEdXInstanceDeploymentStatusSerializer(serializers.Serializer):
     undeployed_changes = serializers.JSONField()
     deployed_changes = serializers.JSONField()
     deployment_type = serializers.ChoiceField(choices=DeploymentType.choices())
+    is_email_verified = serializers.BooleanField()
```

Comment:
The email verification status has nothing to do with a particular deployment. It has to do with a user profile or user's openedx instance. Please move this to "AccountSerializer" and have the API include information about which particular email addresses are unverified.